log.go

```diff
@@ -60,6 +61,7 @@ var (
 	cmgrLog      = backendLog.Logger("CMGR")
 	amgrLog      = backendLog.Logger("AMGR")
 	vspcLog      = backendLog.Logger("VSPC")
+	apiLog       = backendLog.Logger("API")
```

Comment:
Random idea - I'd tag it `3PR` (3rd party requests) or `EXT` (external requests) or something like that to communicate that these are requests outside the P2P network.